internal/vul/scanner.go

```diff
@@ -113,7 +114,7 @@ func (s *imageScanner) Stop() {
 	defer s.mx.RUnlock()
 
 	if s.dbCloser != nil {
```

Comment:
Based on the api changes. It looks like the new api no longer returns a db.Closer. 
So we might as well remove it

Reply:
Removed here and from struct